taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/tasklistener/TaskanaTaskListener.java

```diff
@@ -223,8 +223,16 @@ private String getReferencedTaskJson(DelegateTask delegateTask) throws JsonProce
     referencedTask.setClassificationKey(
         getUserTaskExtensionProperty(delegateTask, "taskana.classification-key"));
     referencedTask.setDomain(getDomainVariable(delegateTask));
-    referencedTask.setWorkbasketKey(getWorkbasketKey(delegateTask));
-    referencedTask.setManualPriority(getManualPriority(delegateTask));
+    referencedTask.setWorkbasketKey(getVariable(delegateTask, "taskana.workbasket-key", null));
+    referencedTask.setManualPriority(getVariable(delegateTask, "taskana.manual-priority", "-1"));
+    referencedTask.setCustomInt1(getVariable(delegateTask, "taskana.customInt1", null));
+    referencedTask.setCustomInt2(getVariable(delegateTask, "taskana.customInt2", null));
+    referencedTask.setCustomInt3(getVariable(delegateTask, "taskana.customInt3", null));
+    referencedTask.setCustomInt4(getVariable(delegateTask, "taskana.customInt4", null));
+    referencedTask.setCustomInt5(getVariable(delegateTask, "taskana.customInt5", null));
+    referencedTask.setCustomInt6(getVariable(delegateTask, "taskana.customInt6", null));
+    referencedTask.setCustomInt7(getVariable(delegateTask, "taskana.customInt7", null));
+    referencedTask.setCustomInt8(getVariable(delegateTask, "taskana.customInt8", null));
```

Comment:
i'd replace the variable name to `custom-int-X` to match the same pattern as workbasket-key and manual-priority